ui-components/src/headers/PageHeader.docs.mdx

```diff
@@ -0,0 +1,45 @@
+import { Canvas, Story, ArgsTable } from "@storybook/addon-docs"
```

Comment:
I think ideally we use the autogenerated docs where possible, are we able to include anything custom here and the autogenerated props if we stick by the Storybook docs format?

Reply:
The `ArgsTable` component provides the auto-gen props docs, so that should be fine. However, I don't know of a way to auto-gen docs on the CSS variables specifically, hence the manual addition of the Markdown content for that. Would love your feedback that approach or if there's a better way to tackle it.

Reply:
sounds good to me!